v2/samples/azureiotedge-simulated-temperature-sensor/DesiredProperties.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Azure.Devices.Shared;
+using Newtonsoft.Json;
+
+namespace azureiotedge_simulated_temperature_sensor
+{
+    public class DesiredProperties
+    {
+        private bool _sendData = true;
+        private int _sendInterval = 5;
+
+        public void UpdateDesiredProperties(TwinCollection twinCollection)
```

Comment:
Nit: UpdateDesiredProperties name can be a bit confusing - it is used to update desired properties in the twin. Suggest making this type immutable and initialized via the ctor. And Program.cs can contain a field of DesiredProperties which can be updated with the latest value.

Reply:
If I make the class immutable I have to recreate it each time a module twin update fires.